File: src/java/eval/KoKoListValue.java
/*
	@author loriacarlos@gmail.com
*/
package kokoslan.ast;
import java.util.*;
import java.io.*;

public class KoKoListValue extends ArrayList<KoKoValue> implements KoKoValue{
	public KoKoListValue(List<KoKoValue> list){
		super(list);
	}
	public KoKoListValue(){
		super();
	}
}

File: src/kotlin/eval/KoKoListValue.kt
/* 
Autores:
	Alejandro Calderon Acuña.
	Linsey Garro Le Roy.
	Angel Gómez Oviedo.

 */
 
package kokoslan.kotlin.ast

import java.util.*
import java.io.*

class KoKoListValue(var list:MutableList<KoKoValue> = mutableListOf()):ArrayList<KoKoValue>(list), KoKoValue